### ISSUE TITLE
influx_hud_hideplayers_wepsnd - Problem (error) - TE_ReadNum ; Line 94

### ISSUE THREAD

**Issue by Fastmancz** [TotallyMehis/Influx-Timer#77]
Hello,

beforehand thanks for your plugin. Sometimes i saw in a console error message: "influx_hud_hideplayers_wepsnd.smx (problem with TE_ReadNum) ; (on the Line 94) ". I don't know what kind of situation that causes this the problem.

Maybe this issue is resolved in a new update, I don't know.

Thanks for your fix.
Regards
Fastmancz, CEO CMG Portal.

Edit:

```
L 02/08/2018 - 17:40:22: [influx_recsounds.smx] [Influx] Sound file 'sound/bot\good_one.wav' does not exist! Ignoring it.
L 02/08/2018 - 17:40:22: [influx_recsounds.smx] [Influx] Sound file 'sound\bot\nice_one_sir.wav' does not exist! Ignoring it.
L 02/08/2018 - 17:40:22: [influx_recsounds.smx] [Influx] Sound file 'sound\bot\owned.wav' does not exist! Ignoring it.

L 02/08/2018 - 18:15:53: [SM]   [1] Line 94, C:\Users\martin\Desktop\BHOP\influx_v1_1_1hotfix_bhop\addons\sourcemod\scripting\influx_hud_hideplayers_wepsnd.sp::E_ShotgunShot
L 02/08/2018 - 18:15:54: [SM] Exception reported: Temp entity property "m_iWeaponID" not found
L 02/08/2018 - 18:15:54: [SM] Blaming: influx_hud_hideplayers_wepsnd.smx
L 02/08/2018 - 18:15:54: [SM] Call stack trace:
L 02/08/2018 - 18:15:54: [SM]   [0] TE_ReadNum
L 02/08/2018 - 18:15:54: [SM]   [1] Line 94, C:\Users\martin\Desktop\BHOP\influx_v1_1_1hotfix_bhop\addons\sourcemod\scripting\influx_hud_hideplayers_wepsnd.sp::E_ShotgunShot
L 02/08/2018 - 18:15:56: [SM] Exception reported: Temp entity property "m_iWeaponID" not found
L 02/08/2018 - 18:15:56: [SM] Blaming: influx_hud_hideplayers_wepsnd.smx
L 02/08/2018 - 18:15:56: [SM] Call stack trace:
L 02/08/2018 - 18:15:56: [SM]   [0] TE_ReadNum
L 02/08/2018 - 18:15:56: [SM]   [1] Line 94, C:\Users\martin\Desktop\BHOP\influx_v1_1_1hotfix_bhop\addons\sourcemod\scripting\influx_hud_hideplayers_wepsnd.sp::E_ShotgunShot
L 02/08/2018 - 18:16:53: [SM] Exception reported: Temp entity property "m_iWeaponID" not found
L 02/08/2018 - 18:16:53: [SM] Blaming: influx_hud_hideplayers_wepsnd.smx
L 02/08/2018 - 18:16:53: [SM] Call stack trace:
```

**Comment by TotallyMehis** [comment]
Fixed in aa74179c17991741e2cda6aa282fd2e5b53a61ec

Can be downloaded here: https://github.com/TotallyMehis/Influx-Timer/blob/dev/addons/sourcemod/scripting/influx_hud_hideplayers_wepsnd.sp

**Comment by Fastmancz** [comment]
Ok, thanks. Solved. 👍🤗
